src/main/webapp/logged_in_homepage.html

```diff
@@ -121,6 +121,5 @@ <h5 class="modal-title" id="newMatchModalLabel">No New Matches Found. Please try
     <script src="vendor/jquery/jquery.min.js"></script>
     <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
     <script src="ts/matches.js"></script>
-    <script src="ts/login.js"></script>
```

Comment:
I think @samuelsandoval1 was trying to include this to implement a redirect. You might need to do some work to ensure the redirect can work.

Reply:
We discussed earlier and decided it makes more sense to just have the logout button on the main page :)